cpp/ps0000/ps0000_soln.cpp

```diff
@@ -1,12 +1,10 @@
-﻿
-#include <iostream>
+﻿#include <iostream> // 2/2
 
 using namespace std;
 
-int main()
+int main() // 1/1

```

Comment:
for `void` it's typeless keyword for function declaration and it's used when function don't take any parameters. Here we use `main` function with type `int` but without any parameters